### PR TITLE
[HIVEMALL-217] Resolve missing links for user manual

### PR DESCRIPTION
**Pull request by chezou**
## What changes were proposed in this pull request?

Fix missing links and unintended redirection on the document.
- Resolve unintended redirects
- Use https insted of http if possible
- Add instruction for KDD Cup 2012 evaluation code

There are still known issues required to be fixed:
- Change of Kaggle documents loses good refere for [Log Loss](https://www.kaggle.com/wiki/LogarithmicLoss) and [Metrics](https://www.kaggle.com/wiki/Metrics) for evaluation for regression page
- Due to version up of EMR version, [this link](https://docs.aws.amazon.com/ElasticMapReduce/latest/DeveloperGuide/emr-plan-ami.html) redirects unintended page
- Mix server related documents are outdated; tips/mixserver, tips/hadoop_tuning

## What type of PR is it?

Improvement

## What is the Jira issue?

https://issues.apache.org/jira/projects/HIVEMALL/issues/HIVEMALL-217

## How was this patch tested?

manual tests

### CODE REVIEW

File: docs/gitbook/tips/hadoop_tuning.md [hunk 1]
```diff
@@ -75,13 +75,13 @@ feature_dimensions (2^24 by the default) * 4 bytes (float) * 2 (iff covariance i
 ```
 > 2^24 * 4 bytes * 2 * 1.2 ≈ 161MB
 
-When [SpaceEfficientDenseModel](https://github.com/apache/incubator-hivemall/blob/master/src/main/java/hivemall/io/SpaceEfficientDenseModel.java) is used, the formula changes as follows:
+When [SpaceEfficientDenseModel](https://github.com/myui/hivemall/blob/master/src/main/java/hivemall/io/SpaceEfficientDenseModel.java) is used, the formula changes as follows:
```
Comment (myui) [inline docs/gitbook/tips/hadoop_tuning.md]: `github.com/myui` is deprecated. 

Use https://github.com/apache/incubator-hivemall/blob/master/core/src/main/java/hivemall/model/SpaceEfficientDenseModel.java instead

other appearance of `github.com/myui` as well.

Comment (chezou) [reply docs/gitbook/tips/hadoop_tuning.md]: I unintenntionay injected this modification. Is it OK to replace from `github.com/myui` into `apache/incubator-hivemall` if there is no redirect target? e.g. `bin/run_mixserv.sh` is still missing.

File: docs/gitbook/tips/mixserver.md [hunk 2]
```diff
@@ -32,8 +33,8 @@ Running Mix Server
 ===================
 
 First, put the following files on server(s) that are accessible from Hadoop worker nodes:
-* [target/hivemall-mixserv.jar](https://github.com/apache/incubator-hivemall/releases)
-* [bin/run_mixserv.sh](https://github.com/apache/incubator-hivemall/raw/master/bin/run_mixserv.sh)
+* [target/hivemall-mixserv.jar](https://github.com/myui/hivemall/releases)
+* [bin/run_mixserv.sh](https://github.com/myui/hivemall/raw/master/bin/run_mixserv.sh)
```
Comment (chezou) [inline docs/gitbook/tips/mixserver.md]: @myui It seems there are no `hivemall-mixserv.jar` and `bin/run_mixserv.sh` in current repo. It'd be nice you to suggest alternative link. Or, I will keep current description.

File: docs/gitbook/tips/hadoop_tuning.md (hunk 1 above)
Comment (chezou) [reply docs/gitbook/tips/hadoop_tuning.md]: OK, fixed github.com/myui as much as I can. https://github.com/apache/incubator-hivemall/pull/162/commits/774f75e8daa83ccd6295be903f08877dcd0c8f52

**Comment by chezou** [comment]
Thanks for review!

**Pull request closed without merging**